docs/api/portable-stories.md

```diff
@@ -0,0 +1,184 @@
+| parameters | `Record<string, any>`                      | The story's parameters                      |
+| load       | `() => Promise<void>`                      | Executes all the loaders for a given story  |
+| play       | `(context) => Promise<void>  \| undefined` | Executes the play function of a given story |
+
+## composeStory
+
+You can use `composeStory` if you wish to compose a single story for a component.
+
+```tsx
+// Button.test.ts|tsx
+import { render, screen } from '@testing-library/react';
+import { composeStory } from '@storybook/react';
+import { vi, test, expect } from 'vitest';
+import meta, { Primary as PrimaryStory } from './Button.stories';
+
+test('onclick handler is called', () => {
+  // Returns a component that already contain all decorators from story level, meta level and global level.
+  const Primary = composeStory(PrimaryStory, meta);
+
+  const onClickSpy = vi.fn();
+  render(<Primary onClick={onClickSpy} />);
+  const buttonElement = screen.getByRole('button');
+  buttonElement.click();
+  expect(onClickSpy).toHaveBeenCalled();
+});
+```
+
+### Type
+
+<!-- prettier-ignore-start -->
+```ts
+(
+  story: Story export,
+  componentAnnotations: Meta,
+  projectAnnotations?: ProjectAnnotations,
+  exportsName?: string
+) => ComposedStoryFn
+```
+<!-- prettier-ignore-end -->
+
+### Parameters
+
+#### `story`
+
+(**Required**)
+
+Type: `Story export`
+
+Specifies which story you want to compose.
+
+#### `componentAnnotations`
+
+(**Required**)
+
+Type: `Meta`
+
+The default export from the stories file containing the [`story`](#story).
+
+#### `projectAnnotations`
+
+Type: `ProjectAnnotation | ProjectAnnotation[]`
+
+This parameter is provided for convenience. You should likely use [`setProjectAnnotations`](#setprojectannotations) instead. Details about the `ProjectAnnotation` type can be found in that function's [`projectAnnotations`](#projectannotations-2) parameter.
+
+#### `exportsName`
+
+Type: `string`
+
+TK - What is this?
+
+### Return
+
+Type: `ComposedStoryFn`
+
+A single [composed story](#return).
+
+## setProjectAnnotations
+
+This API should be called once and before the tests run. This will make sure that whenever `composeStories` or `composeStory` are used, the project annotations are taken into account as well.
+
+```ts
+// setup-tests.ts
+import { setProjectAnnotations } from '@storybook/react';
+import * as frameworkAnnotations from '@storybook/nextjs/preview';
+import * as addonAnnotations from 'my-addon/preview';
+import * as previewAnnotations from './.storybook/preview';
+
+setProjectAnnotations([previewAnnotations, frameworkAnnotations, addonAnnotations]);
+```
+
+### Type
+
+```ts
+(projectAnnotations: ProjectAnnotation | ProjectAnnotation[]) => void
+```
+
+### Parameters
+
+#### `projectAnnotations`
+
+(**Required**)
+
+Type: `ProjectAnnotation | ProjectAnnotation[]`
+
+TK - Describe annotations (maybe have a dedicated section at the bottom that this references?)
```

Comment:
I agree, and in fact we don't mention the term `annotation` properly throughout the docs, so users might not understand this straight away. The only documentation about project annotations I think we could refer to is this:
https://storybook.js.org/docs/configure#configure-story-rendering

Reply:
But I think you could mention that projectAnnotations relate to the `.storybook/preview.ts` file, so users can define overrides that would match the file signature